chainer/training/extensions/parameter_statistics.py

```diff
@@ -38,6 +39,9 @@ class ParameterStatistics(extension.Extension):
         prefix (str): Optional prefix to prepend to the report keys.
         trigger: Trigger that decides when to aggregate the results and report
             the values.
+        skip_nan_params (bool): If ``True``, statistics are not computed for
+            parameters including NaNs and a single NaN value is immediately
+            reported instead.
```

Comment:
Added more descriptive text.

Reply:
Ah, I mean, please state that `False` is assumed by default.

Reply:
I have purposely left it out (and similarly for other arguments) since it is duplicate information with the function definition(?). Although I agree it sometimes helps in terms of readability. Do you still think we should add it?

Reply:
Ah, I see.
You're right, and it seems in many other places default value is not necessarily explicitly stated.
Thank you for pointing out.